src/components/history-cmp/HistoryCmp.res

```diff
@@ -0,0 +1,25 @@
+%%raw("import './HistoryCmp.css'")
+@react.component
+let make = (~transactions: array<TransactionType.t>) => {
+    let notEmpty = (x: TransactionType.t) => (x.text != "" && x.amount != 0.)
```

Comment:
This is not the validation for taking input. Form input is validated separately it wont' take blank inputs. This is to filter out the initial state of the transaction which is a blank transaction, so that it wont' show up in our History Component.